adapters/csv_user_repository_adapter.go

```diff
@@ -0,0 +1,37 @@
+package adapters
+
+import (
+	"github.com/iszandro/golang-bootcamp-2020/models"
+	"github.com/iszandro/golang-bootcamp-2020/repositories"
+)
+
+// CSVUserRepositoryAdapter allows to adapt repositories.CVSUserRepository into a
+// interfaces.UserRepository interface.
+// Due to repositories.CVSUserRepository needs a file path for its GetUsers(filePath string) method,
+// this adapter responds to GetUsers() method by setting file path.
+// If FilePath is empty, defaultUsersPath constant is used.
+type CSVUserRepositoryAdapter struct {
+	csvUserRepository repositories.CVSUserRepository
+	FilePath          string
+}
+
+const defaultUsersPath = "fixtures/files/users.csv"
+
+func NewCSVUserRepositoryAdapter() CSVUserRepositoryAdapter {
+	return CSVUserRepositoryAdapter{
+		csvUserRepository: repositories.CVSUserRepository{},
+	}
+}
```

Comment:
You are initializing a `CSVUserRepositoryAdapter` with no values received in the function.

The idea here is to be able to create an instance with the specific data in `CSVUserRepositoryAdapter` struct.

Something like this:
```suggestion
func NewCSVUserRepositoryAdapter(csvUserRepository repositories.CVSUserRepository, filePath string) CSVUserRepositoryAdapter {
	return CSVUserRepositoryAdapter{
		csvUserRepository: csvUserRepository,
		Filepath: filepath,
	}
}
```